css/calculator.css

```diff
@@ -0,0 +1,42 @@
+.calculator{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: black;
+  width: 280px;
+  margin: auto;
+  border-radius: 16px;
+  padding: 16px;
+}
+#result{
+  font-size: 48px;
+  height: 54px;
+  padding: 16px;
+  text-align: end;
+  color: white;
+}
+button{
+  padding: 8px;
+  width: 64px;
+  height: 64px;
+  font-size: 24px;
+  border: 1px solid white;
+  border-radius: 50%;
+  color: #e8eaed;
+  cursor: pointer;
+}
+
+.number-button{
```

Comment:
se for fazer um modificador de um nome de uma classe, use a seguinte convenção:

```button--number```

Esse artigo aqui fala um pouco sobre o BEM CSS https://medium.com/trainingcenter/bem-em-5min-f5c80fd23439